mcd/mcd-deplyoment/mcd-deployment.md

```diff
@@ -0,0 +1,309 @@
+        ]
+      },
+      "ilks": {
+        "A": {
+          "mat": "<Liquidation ratio value in percentage (e.g. 150)>",
+          "line": "<Debt ceiling value in DAI unit (won't be used if autoLine is > 0)>",
+          "autoLine": "<Max debt ceiling value in DAI unit (for DssAutoLine IAM)>",
+          "autoLineGap": "<Value to set the ceiling over the current ilk debt in DAI unit (for DssAutoLine IAM)>",
+          "autoLineTtl": "<Time between debt ceiling increments (for DssAutoLine IAM)>",
+          "dust": "<Min amount of debt a CDP can hold in DAI unit>"
+          "duty": "<Collateral component of stability fee in percentage per year (e.g. 2.5)>",
+          "flipDeploy": {
+            "chop": "<Liquidation penalty value in percentage (e.g. 12.5)>",
+            "dunk": "<Liquidation Quantity in DAI Unit>",
+            "beg": "<Minimum bid increase in percentage (e.g. 5.5)>",
+            "ttl": "<Max time between bids in seconds>",
+            "tau": "<Max auction duration in seconds>"
+          },
+          "clipDeploy": { // Will be used only if there isn't a flipDeploy
+            "chop": "<Liquidation penalty value in percentage (e.g. 12.5)>",
+            "hole": "<Max DAI needed to cover debt+fees of active auctions per ilk (e.g. 100,000 DAI)>",
+            "chip": "<Percentage of due to suck from vow to incentivize keepers (e.g. 2%)>",
+            "tip": "<Flat fee to suck from vow to incentivize keepers (e.g. 100 DAI)>",
+            "buf": "<Multiplicative factor to increase starting price (e.g. 125%)>",
+            "tail": "<Time elapsed before auction reset in seconds>",
+            "cusp": "<Percentage taken for the new price before auction reset (e.g. 30%)>",
+            "calc": {
+              "type": "LinearDecrease/StairstepExponentialDecrease/ExponentialDecrease",
+              "tau":  "<Time after auction start when the price reaches zero in seconds (LinearDecrease)>",
+              "step": "<Length of time between price drops in seconds (StairstepExponentialDecrease)>",
+              "cut":  "<Percentage to be taken as new price per step (e.g. 99%, which is 1% drop) (StairstepExponentialDecrease/ExponentialDecrease)>"
+            },
+            "cm_tolerance": "<Percentage of previous price which a drop would enable anyone to be able to circuit break the liquidator via ClipperMom (e.g. 50%)>"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+<div style="padding: 15px; border: 1px solid transparent; border-color: transparent; margin-bottom: 20px; border-radius: 4px; color: #8a6d3b;; background-color: #fcf8e3; border-color: #faebcc;">
+<h5 style="color: #8a6d3b">Note</h5>
+Make sure to launch <bold style="color: grey">MCD_ADMIN</bold> if you are providing it in <bold style="color: grey">config.authority</bold>.
+</div>
+```shell
+sethSend "$MCD_GOV" 'mint(address,uint256)' "$ETH_FROM" "$(seth --to-uint256 "$(seth --to-wei 1000000 ETH)")"
+```
```

Comment:
```suggestion

```bash
sethSend "$MCD_GOV" 'mint(address,uint256)' $ETH_FROM $(seth --to-wei 1000000 ETH)
```
```